container-host-files/opt/scf/bin/klog.sh

```diff
@@ -5,192 +5,82 @@ set -e
 KLOG=${HOME}/klog
 
 if [ "$1" == "-h" ]; then
-    cat <<EOF
+	cat <<EOF
```

Comment:
Not sure changing to tab indents (and other random stylistic changes) in an unrelated commit is a good idea.

Reply:
For now, will revert indentation related changes and handle them in a separate commit.